src/frontend/specification/StenoGraphs.jl

```diff
@@ -4,6 +4,8 @@
 ### Define Modifiers
 ############################################################################################
 
+AbstractStenoGraph = AbstractVector
```

Comment:
How can I understand this line? ^^

Reply:
You are right, I should have probably added an explanation there:
`// FIXME: remove when StenoGraphs.jl will provide AbstractStenoGraph`

The reason is -- in this PR I switch constructors of *SemSpecification* objects from keyword arguments to positional arguments, which enables types-based dispatch (it simplifies the code a bit and makes it more type-agnostic).
It requires that input types for different ctors are unambiguously distinct (*ParameterTable*, *StenoGraph*, *RAMMatrices*).
*StenoGraphs.jl* does not provide its own graph type, but luckily the vector of edges (which is the current representation of the graph in *StenoGraphs.jl*) is unambiguous.

Reply:
Makes sense - I opened an issue in `StenoGraphs`. In the meantime, maybe something like this is a bit more precise:
```suggestion
#FIXME: remove when StenoGraphs.jl will provide AbstractStenoGraph
const AbstractStenoGraph = AbstractArray{T, 1} where {T <: StenoGraphs.AbstractEdge}
```